app/src/pages/movies/movie/components/EditMovieForm.jsx

```diff
@@ -74,13 +81,13 @@ EditMovieForm.propTypes = {
     close: PropTypes.func.isRequired,
     movie: PropTypes.shape({
         id: PropTypes.number.isRequired,
-        name: PropTypes.string.isRequired,
-        age: PropTypes.number.isRequired,
-        genre: PropTypes.string.isRequired,
-        imgSrc: PropTypes.string.isRequired,
+        title: PropTypes.string.isRequired,
+        release_date: PropTypes.string.isRequired,
+        genres: PropTypes.array.isRequired,

```

Comment:
wrote somewhere - it's not ok to use `array` props check

Reply:
I'm sorry, I skipped that) Which type would be better?

Reply:
PropTypes.array -> PropTypes.arrayOf([PropTypes.string])

PropTypes.object-> PropTypes.shape({
   key: Proptypes.string.isRequired
})